src/create.js

```diff
@@ -280,7 +292,28 @@ function getChangesFromFile(core, file, id) {
     core.info(`Detected a change to the data file`);
     const changes = [];
 
-    const sourceChanges = [...file.patch.matchAll(JSON_CHANGE_EXPR)];
+    core.debug(`\nSimple Icons data file`);
+    core.debug(file);
+
+    let filePatch = file.patch;
+
+    if (!filePatch) {
+      const contentResult = await client.rest.repos.getContent({
+        owner: context.repo.owner,
+        repo: context.repo.repo,
+        path: file.filename,
+        ref: file.sha,
+      });
+
+      filePatch = Buffer.from(
```

Comment:
This variable `filePatch` is not being used, note that you lately use `file.patch` to check for changes.

Reply:
The `file.patch` is possibly undefined. See https://github.com/simple-icons/simple-icons/issues/8734#issuecomment-1585995877.